### PR TITLE
Add SPIFFE standard doc, update README

### PR DESCRIPTION
**Pull request by evan2645**
This PR adds a first pass at the core SPIFFE standard, as well as updates the README to point to it. I will create additional PRs as more standardization docs are ready

Works to address part of #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,32 @@
...
+* Perform node and workload attestation
+
+* Implement a signing framework for securely issuing and renewing SVIDs
+
+* Provide an API for registering nodes and workloads, along with their designated SPIFFE IDs
 
 # The Project
 
-## The Design
+## The Standards
```
Comment (evan2645) [inline README.md:25]: I figure we can add `## The Reference Implementation` here once we're ready to share something

**Review by drrt — APPROVED** [review]
👍

**Review by y2bishop2y — APPROVED** [review]
LGTM

**Comment by y2bishop2y** [comment]
I put my comments in the google doc. Looks good. One question I have is how are we going to define the WL API. and where are we going to put those definitions

**Pull request merged**